alerts/email_test.go

```diff
@@ -17,13 +17,14 @@ import (
 const (
 	ProjectName     = "Test Project"
 	ProjectOwner    = "testprojowner"
+	ProjectRepo     = "testrepo"
```

Comment:
Noticed this was missing when I went to break things, if it's too distracting, let me know and I'll take it out.